packaging/cpackconfig_nsis.cmake

```diff
@@ -11,6 +11,7 @@ cpack_add_component(arduino DISPLAY_NAME "Arduino Drivers")
 # ========================================
 set(CPACK_NSIS_ENABLE_UNINSTALL_BEFORE_INSTALL ON)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY ".")
+set(CPACK_NSIS_STARTMENU_DIRECTORY "Ultimaker Cura")
```

Comment:
Why not distinguish between Cura and Enterprise here? Is NSIS community edition only?

Reply:
Indeed, NSIS creates .exe installers, which will never be used for Enterprise builds.